cal/api/serializers.py

```diff
@@ -82,3 +82,9 @@ def get_hours(self, obj):
         start = self.context.get('start')
         end = self.context.get('end')
         return obj.hours(calendar_ids=calendar_ids, start=start, end=end)
+
+
+class ColorCategoryList(serializers.ModelSerializer):
```

Comment:
This should be named something with `Serializer` at the end. `List` always refers to a view, and thus causes confusion. Maybe `ColorCategoryTimeSeriesSerializer`?

Was there something like this in the previous PR?